package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "airbnb-browser-shims",
-  "version": "1.5.0",
+  "version": "1.6.0",
```

Comment:
please revert this; version bumps should only ever be in a commit directly to master, not in PRs.